TaxFoundation/facts-and-figures-web
# 2022 facts and figures

# Description

This PR updates the facts and figures to the most current 2022 data as well as removing almost 80 security vulnerabilities.

# Steps to test

1. Run 'npm install'
2. Run 'npm run start'
3. app should compile and launch in browser
4. test that all 42 pages render properly

# Tasks Linked
[862jcyqyt](https://app.clickup.com/t/862jcyqyt)
